liquidcore/config/views.py

```diff
@@ -356,7 +356,9 @@ def parse(self, stream, media_type=None, parser_context=None):
 @permission_classes([IsAdminUser])
 @parser_classes([OVPNParser])
 def vpn_client_upload(request):
-    ovpn_content = request.data
-    # TODO send ovpn content somewhere in a try/except
-    # that handles invalid ovpn files and/or errors
+    ovpn_content = request.data.decode('latin1')
+    setting = Setting.objects.get(name='vpn_client_config_file')
+    setting.data = ovpn_content
```

Comment:
See above: instead of having a `Setting` named `vpn_client_config_file` set to a string with the file content, it should be a dict with a key that says `content`.